coffea/jetmet_tools/CorrectedJetsFactory.py

```diff
@@ -52,11 +61,22 @@ def getfunction(layout, depth, **kwargs):
             )
         return None
 
-    out = awkward.transform(
-        getfunction,
-        item,
-        behavior=item.behavior,
-    )
+    out = None
+    if backend == "cpu":
+        out = awkward.transform(
+            getfunction,
+            item,
+            behavior=item.behavior,
+        )
+    elif backend == "typetracer":
+        zlitem = item.layout.form.length_zero_array()
```

Comment:
We're trying to avoid the need to use `length_zero_array` in typetracer code unless it's 100% needed. Would you be able to keep tabs on places where you're using `length_zero_array` for workarounds so that we can remove them in future?

Reply:
Yes - some of this is going through growing pains as I start using `dask_awkward` as the data source instead of `dak.from_awkward`, the former has rather more intense usage of typetracers.

Once we're on the other side I'll make sure it's minimal and let you know the rest.

Reply:
The one you highlight here will probably stay since I need to feed a non-zero-length array to as a PRNG seed.